docs/op-guide/deploy-tikv-using-docker-compose.md

```diff
@@ -43,8 +43,10 @@ Make sure you have installed the following items on your machine:
 2. Edit the `compose/values.yaml` file to configure `networkMode` to `host` and comment the TiDB section out.
 
     ```bash
-    cd tidb-docker-compose/compose
-    networkMode: host
+    cd tidb-docker-compose
+    vim compose/values.yaml
```

Comment:
The above two lines are unnecessary. Please remove them.

Reply:
I think it is necessary. The two lines tell the users where the yaml file is and how to edit it.